### ISSUE TITLE
convert_to_ucs_python errors out when trying to reboot blade server

### ISSUE THREAD

**Issue by waghswapnil** [CiscoUcs/ucsmsdk#77]
**Console Logs** 

```
In [1]: from ucsmsdk.utils.converttopython import convert_to_ucs_python

In [2]: file_path = "..."

In [3]: convert_to_ucs_python(dump_xml=True, dump_to_file=True, dump_file_path=file_path)
### Please review the generated cmdlets before deployment.

### Script Output is in file < ....>
ucsm logfile: /Users/swapnilwagh/Library/Application Support/Oracle/Java/Deployment/log/.ucsm/centrale_15774.log

---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-3-965d82646705> in <module>()
----> 1 convert_to_ucs_python(dump_xml=True, dump_to_file=True, dump_file_path=file_path)

/Library/Python/2.7/site-packages/ucsmsdk/utils/converttopython.pyc in convert_to_ucs_python(log_path, xml, request, gui_log, path, literal_path, dump_to_file, dump_file_path, dump_xml)
   1519             # if line:
   1520             #     _find_xml_requests_in_file_test(file_stream, True)
-> 1521             _find_xml_requests_in_file(file_stream, True)
   1522             time.sleep(2)
   1523         file_stream.close()

/Library/Python/2.7/site-packages/ucsmsdk/utils/converttopython.pyc in _find_xml_requests_in_file(file_stream, gui_log)
   1314             if line is not None:
   1315                 # print "[%s]" %(line)
-> 1316                 _extract_xml(file_stream, line)
   1317         line = file_stream.readline()
   1318

/Library/Python/2.7/site-packages/ucsmsdk/utils/converttopython.pyc in _extract_xml(file_stream, line)
   1292                 continue
   1293             read_flag = False
-> 1294             _generate_cmdlets(request_string)
   1295             request_string = ""
   1296             break

/Library/Python/2.7/site-packages/ucsmsdk/utils/converttopython.pyc in _generate_cmdlets(xml_string)
   1236
   1237     if category == "configConfMos" or category == "configConfMo":
-> 1238         cmdlet = _generate_config_conf_cmdlets(xml_string)
   1239     elif category in ["configResolveDn", "configResolveDns",
   1240                       "configResolveClass", "configResolveClasses"]:

/Library/Python/2.7/site-packages/ucsmsdk/utils/converttopython.pyc in _generate_config_conf_cmdlets(xml_string)
    604
    605     if pair_nodes is None or len(pair_nodes) < 1:
--> 606         node = top_node.getElementsByTagName("inConfig")[0]
    607         if node is None:
    608             return

IndexError: list index out of range
```

**Comment by waghswapnil** [comment]
**XML Dump**
```
<configConfMos cookie="[hidden]" response="yes">
<outConfigs>
  <pair key="sys/chassis-1/blade-6">
   <computeBlade adminPower="policy" adminState="in-service" assignedToDn="org-root/ls-BS-CAB" association="associated" availability="unavailable" availableMemory="16384" chassisId="1" checkPoint="discovered" connPath="A,B" connStatus="A,B" descr="" discovery="complete" discoveryStatus="" dn="sys/chassis-1/blade-6" fltAggr="12884901888" fsmDescr="" fsmFlags="" fsmPrev="PowercycleSuccess" fsmProgr="100" fsmRmtInvErrCode="none" fsmRmtInvErrDescr="" fsmRmtInvRslt="" fsmStageDescr="" fsmStamp="2016-06-23T10:57:46.729" fsmStatus="PowercycleBegin" fsmTry="0" intId="54032" lc="discovered" lcTs="1970-01-01T00:00:00.000" localId="" lowVoltageMemory="regular-voltage" managingInst="B" memorySpeed="1333" mfgTime="2014-09-20T00:00:00.000" model="UCSB-B200-M3" name="" numOf40GAdaptorsWithOldFw="0" numOf40GAdaptorsWithUnknownFw="0" numOfAdaptors="2" numOfCores="8" numOfCoresEnabled="8" numOfCpus="2" numOfEthHostIfs="2" numOfFcHostIfs="0" numOfThreads="8" operPower="on" operPwrTransSrc="software_mcserver" operQualifier="" operState="ok" operability="operable" originalUuid="e25d9a8f-5c85-4ec1-96aa-f38cb331a946" partNumber="73-14689-04" policyLevel="0" policyOwner="local" presence="equipped" revision="0" scaledMode="none" serial="FCH1837JDYP" serverId="1/6" slotId="6" status="modified" totalMemory="16384" usrLbl="" uuid="00000000-0000-0000-0000-000000024545" vendor="Cisco Systems Inc" vid="V06"/>
  </pair>
 </outConfigs>
</configConfMos>


```

**Comment by vvb** [comment]
@waghswapnil thanks for reporting the issue.. will take a look

**Comment by vvb** [comment]
@waghswapnil this seems to be because of the message being a secure method. As we discussed, you should try and launch the GUI using the sdk, and not directly from browser.

**Comment by waghswapnil** [comment]
@vvb after updating to the latest sdk, the converttopython script is stuck and it doesn't dump the xml.
Following is the traceback :- 

In [8]: convert_to_ucs_python(dump_xml=True, dump_to_file=True, dump_file_path=file_path)
### Please review the generated cmdlets before deployment.

### Script Output is in file < /Users/swapnilwagh/Documents/WindowsData/MyDocuments/Data/Workspace/UCSM/PythonSdk/temp-files/log >
ucsm logfile: /Users/swapnilwagh/Library/Application Support/Oracle/Java/Deployment/log/.ucsm/centrale_9725.log

Exception in thread Thread-1:
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 1082, in run
    self.function(*self.args, **self.kwargs)
  File "ucsmsdk/ucssession.py", line 398, in _refresh
    self.__password)
  File "ucsmsdk/ucsmethodfactory.py", line 123, in aaa_refresh
    method = ExternalMethod("AaaRefresh")
  File "ucsmsdk/ucsmethod.py", line 51, in __init__
    meta_module = ucscoreutils.load_module(method_id)
  File "ucsmsdk/ucscoreutils.py", line 114, in load_module
    [module_name])
ImportError: No module named AaaRefreshMeta


^C---------------------------------------------------------------------------
KeyboardInterrupt                         Traceback (most recent call last)
<ipython-input-8-965d82646705> in <module>()
----> 1 convert_to_ucs_python(dump_xml=True, dump_to_file=True, dump_file_path=file_path)

/Library/Python/2.7/site-packages/ucsmsdk-0.9.1.0-py2.7.egg/ucsmsdk/utils/converttopython.py in convert_to_ucs_python(log_path, xml, request, gui_log, path, literal_path, dump_to_file, dump_file_path, dump_xml)
   1520             #     _find_xml_requests_in_file_test(file_stream, True)
   1521             _find_xml_requests_in_file(file_stream, True)
-> 1522             time.sleep(2)
   1523         file_stream.close()
   1524

KeyboardInterrupt:

**Comment by ragupta-git** [comment]
@waghswapnil : Could you please share the ucsm logfile.

**Comment by waghswapnil** [comment]
@ragupta-git confirmed that the logging of secured property does not get enabled when using `log.show.encrypted` in java version `1.6.0_65`.

```
2016-06-29 15:09:41,872 - ucs - DEBUG - Java Version: <1.6.0_65>
2016-06-29 15:09:41,873 - ucs - DEBUG - Enable Log String is 	<property name="log.show.encrypted" value="true"/>.
```

**Comment by vvb** [comment]
@ragupta-git we need to evaluate if the property name to be used is now different above a 1.6.x version within the 1.6.x series..  Similar to 1.7.x {x < 45, uses `log.show.encrypted`.. x >= 45 something else}

**Comment by vvb** [comment]
resolved by #84
